addons/mail/models/mail_thread.py

```diff
@@ -2005,9 +2031,10 @@ def _message_log(self, *, body='', author_id=None, email_from=None, subject=Fals
             'record_name': False,
             'reply_to': self.env['mail.thread']._notify_get_reply_to(default=email_from)[False],
             'message_id': tools.generate_tracking_message_id('message-notify'),  # why? this is all but a notify
+            'add_sign': False,
```

Comment:
Would it be a bad idea to support it with:

`kwargs.get('add_sign', False)`

Or do we never ever want the signature when using message_log ?

Reply:
Message_log never sends notifications (as explained in docstring), hence no need to support any signature.